src/issues/currentIssue.ts

```diff
@@ -130,19 +129,30 @@ export class CurrentIssue extends Disposable {
 		return undefined;
 	}
 
-	private async createOrCheckoutBranch(branch: string): Promise<boolean> {
+	private async createOrCheckoutBranch(branch: string, silent: boolean): Promise<boolean> {
```

Comment:
Does this really need a `silent`? It seems like something that would always be useful to know if it failed.